pkgs/games/gemrb/default.nix

```diff
@@ -1,32 +1,71 @@
-{ lib, stdenv, fetchFromGitHub, cmake
-, freetype, SDL2, SDL2_mixer, openal, zlib, libpng, python2, libvorbis
-, libiconv }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, SDL2
+, SDL2_mixer
+, freetype
+, libGL
+, libiconv
+, libpng
+, libvlc
+, libvorbis
+, openal
+, python2 # 0.9.0 crashes after character generation with py3, so stick to py2 for now
```

Comment:
Is there an upstream issue about this you can link? We want to get rid of python2 rather sooner than later.

Reply:
There are several - it just looks like py3 support isn't quite ready yet. When the next version is released, I'll give it another go with py3.